sso-kit-starter/src/main/java/com/vaadin/sso/starter/SingleSignOnConfiguration.java

```diff
@@ -145,10 +145,10 @@ protected void configure(HttpSecurity http) throws Exception {
         });
     }
 
-    static class DefaultVaadinAuthContext implements VaadinAuthContext {
+    static class DefaultVaadinAuthContext implements AuthenticationContext {
```

Comment:
Shouldn't the class be also renamed to something like `DefaultAuthenticationContext`?
Same thing would apply to the property name defined on line `61`.